test/controllers/agent/ConfirmClientVrnControllerSpec.scala

```diff
@@ -181,4 +165,48 @@ class ConfirmClientVrnControllerSpec extends ControllerBaseSpec with MockCustome
       }
     }
   }
+
+  "Calling the .redirectToSessionUrl action" when {
+
+    "the user is an Agent" when {
+
+      "the Agent is authorised and signed up to HMRC-AS-AGENT" when {
```

Comment:
I reckon we don't need this test clause as you don't have or need a test to check that the agent does/doesn't have the enrolment.